arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json

```diff
@@ -0,0 +1,691 @@
+            "description": "The parameters to provide for the created account."
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "If resource is created successfully, the service should return 200 (OK).",
+            "schema": {
+              "$ref": "#/definitions/CognitiveServicesAccount"
+            }
+          },
+          "201": {
+            "description": "If resource is created successfully, the service should return 200 (OK).",
+            "schema": {
+              "$ref": "#/definitions/CognitiveServicesAccount"
+            }
+          },
+          "400": {
+            "description": "If kind or sku.name is invalid, the service should return 400 (Bad Request)."
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "CognitiveServicesAccounts"
+        ],
+        "description": "Update tags or SKU.",
+        "operationId": "CognitiveServicesAccounts_Update",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group within the user's subscription."
+          },
+          {
+            "name": "accountName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the cognitive services account within the specified resource group. Cognitive Services account names must be between 3 and 24 characters in length and use numbers and lower-case letters only.",
+            "maxLength": 24,
+            "minLength": 3
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CognitiveServicesAccountUpdateParameters"
+            },
+            "description": "The parameters to provide for the created account."
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The resource provider should return 200 (OK) to indicate that the operation completed successfully. ",
+            "schema": {
+              "$ref": "#/definitions/CognitiveServicesAccount"
+            }
+          },
+          "404": {
+            "description": "If the resource group *or* resource does not exist, 404 (NotFound) should be returned."
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "CognitiveServicesAccounts"
+        ],
+        "description": "Deletes a resource from the resource group. ",
```

Comment:
"Deletes a ~~resource~~ CognitiveServices Account from the resource group." (remove the empty space after period in the end).